### PR TITLE
Adding select-prompter.

### PR DESCRIPTION
**Pull request by VyacheslavIsaev**
Select prompter allows to select a value from a list of predefined values and transform code depends on the selected value. For example: it allows include one of 3 provided implementations depends on the choice.

### CODE REVIEW

**Review by titusjaka — APPROVED** [review]
LGTM 👍

**Comment by VyacheslavIsaev** [comment]
My apologies, accidentally clicked in a wrong place of the page and exactly hit re-review button...

**Review by titusjaka — APPROVED** [review]

File: inputs/cli_test.go [hunk 1]
```diff
@@ -53,3 +53,24 @@ func TestParseCLIArgsInputs(t *testing.T) {
...
+		ID:      "s",
+		Options: []string{"simple", "advanced"},
+	})
+
+	err := ParseCLIArgsInputs(&mockInputsCollector{prompters: []Prompter{p}}, []string{})
+	require.NoError(t, err)
+	assert.False(p.Answered)
+
+	err = ParseCLIArgsInputs(&mockInputsCollector{prompters: []Prompter{p}}, []string{"--s", "simple"})
+	require.NoError(t, err)
```
Comment (rantav) [inline inputs/cli_test.go:70]: These two likes seem redundant given the four lines below, they test the same thing

**Review by rantav — CHANGES_REQUESTED** [review]
Thanks for doing this, I added a few more suggestions. I see it's already merged but please consider creating a followup MR to fix these

File: inputs/cli_test.go (hunk 1 above)
Comment (VyacheslavIsaev) [reply inputs/cli_test.go:70]: That supposed to be an error test. Probably got distracted after Copy-Paste.

**Pull request merged**